Receipt Link and  Fiscal References (#92)
snabble/docs: api_orders.md

}
```

### Fiscal Reference

> The Fiscal Reference provides the metadata and logs that where

Reviewer: ```suggestion
The Fiscal Reference provides the metadata and logs that were
```

> obtained by required fiscall processes. The provided information and

Reviewer: ```suggestion
obtained by required fiscal processes. The provided information and
```

> carried out steps heavily depent on the place where the transaction

Reviewer: ```suggestion
carried out steps heavily depend on the country where the transaction
```

was executed an the modus of operation.

> For germany the process is regulated by the Kassensicherungsverordnung

Reviewer: ```suggestion
In germany the process is regulated by the Kassensicherungsverordnung
```

(KassenSichV) and hence the values are provided by a Technische
Sicherheitseinrichtung (TSE).

| Path                                      | Type     | Description                                                |
|-------------------------------------------|----------|------------------------------------------------------------|
| `securityDeviceModule.serialNumber`       | `string` | Serial number of the security device                       |
| `securityDeviceModule.signatureAlgorithm` | `string` | Signature Algorithm used by the device                     |
| `securityDeviceModule.logTimeFormat`      | `string` | Time Format of the security device                         |
| `securityDeviceModule.certificate`        | `string` | Certificate presented by the security device               |
| `logs[].timestamp`                        | `string` | RFC3339 encoded point in time on which the log was created |
| `logs[].counter`                          | `int`    | Counter value of the signature                             |
| `logs[].processType`                      | `string` | The process type send to the security device               |

> | `logs[].processData`                      | `string` | The data send to to security device                        |

Reviewer: ```suggestion
| `logs[].processData`                      | `string` | The data send to security device                        |
```
